index.js

```diff
@@ -115,10 +118,49 @@ module.exports = {
 
         return app;
     },
+
+    // Outputs all pod scss files into the style tree but prefixed with ember-osf
+    // This allows apps using this addon to import all the scss they want using "@import 'ember-osf'"
+    // The actual 'ember-osf' namespace is exported by app/styles/_ember-osf.scss
```

Comment:
This (again) feels like we're reinventing ember-component-css. It's possible there's a good reason for that, but I suspect that good reason may have expired a year ago.